src/PAModel/Schemas/LocalResourceInfo.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.Formulas.Tools.Schemas
+{
+    internal class LocalAssetInfoJson
```

Comment:
Add a comment calling out that this schema is only  for the unpacked side, not part of the .msapp